invenio_rdm_records/resources/serializers/marcxml/schema.py

```diff
@@ -286,3 +288,58 @@ def get_formats(self, obj):
         """Get data formats."""
         formats = obj["metadata"].get("formats", missing)
         return formats
+
+    @post_dump()
+    def changeKeysToTags(self, data, many, **kwargs):
```

Comment:
nit: I know it's out of scope for the change, but is there a reason this is camelCased?